modules/microshift-embed-ostree-enable-eus-repos.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * microshift_install/microshift-embed-into-rpm-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enable-eus-repos_{context}"]
+= Enabling extended support repositories for image building
+
+If you have an extended support (EUS) release of {microshift-short}, you must enable the {op-system-base-full} EUS repositories for image builder to use. If you do not have an EUS version, you can skip these steps.
+
+.Prerequisites
+
+* You have an EUS version of {microshift-short} or are updating to one.
+* You have root-user access to your build host.
+* You reviewed the link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.14/html/installing/microshift-install-rpm#install-rpm-rhde-compatibility-table[{op-system-bundle} release compatibility matrix].
+
+include::snippets/microshift-unsupported-config-warn.adoc[leveloffset=+1]
+
+.Procedure
+
```

Comment:
I updated the nit's sheet with this information, as well but there is a missing initial step.
1. Create /etc/osbuild-composer/repositories
sudo mkdir -p /etc/osbuild-composer/repositories
2. Copy /usr/share/osbuild-composer/repositories/rhel-9.2.json into /etc/osbuild-composer/repositories
sudo cp /usr/share/osbuild-composer/repositories/rhel-9.2.json /etc/osbuild-composer/repositories/rhel-9.2.json